### PR TITLE
feat: support sveltekit-i18n

### LINKED ISSUE #44: Ad i18n and l10n
[`sveltekit-i18n`](https://github.com/sveltekit-i18n/lib)

I can offer support for :

* English
* Bangla
* Japanese 
* Hindi
* Urdu

### PR DESCRIPTION
**Pull request by tokitouq**
Closes #44

### CODE REVIEW

File: src/routes/+layout.server.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { locales, loadTranslations, translations, defaultLocale } from "$lib/translations";
+import type { ServerLoad } from "@sveltejs/kit";
+import type { Translations } from "../../node_modules/sveltekit-i18n/node_modules/@sveltekit-i18n/base";
```
Comment (tokitouq) [inline src/routes/+layout.server.ts:3]: I've made an issue regarding this:
https://github.com/sveltekit-i18n/lib/issues/181

**Comment by baseplate-admin** [comment]
I have added some translation. Most of them sound like i am a 1st grader. Its a start. you make the switching logic, i will add more configs

**Comment by tokitouq** [comment]
> Its a start. you make the switching logic, i will add more configs

Yea I will do that, also I will try to expose more keys.
I will update you, until then no work on this okay?

**Comment by tokitouq** [comment]
@baseplate-admin what do you think bout adding other languages support later? I mean after testing with some langs?
Also I used AI to generate bengali translations, see if it looks good to you. if so we can use it :zap:

File: src/lib/translations/bn/home.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+	"sidebar": {
+		"home": "হোম",
+		"explore": "অন্বেষণ",
```
Comment (baseplate-admin) [inline src/lib/translations/bn/home.json:4]: অন্বেষণ -> খুঁজুন

File: src/lib/translations/bn/home.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+		"list": "তালিকা",
+		"schedule": "সময়সূচী",
+		"forum": "ফোরাম",
```
Comment (baseplate-admin) [inline src/lib/translations/bn/home.json:7]: ফোরাম -> মঞ্চ

**Review by baseplate-admin — CHANGES_REQUESTED** [review]
This is unscalable. we want new translation backend @akindworld

File: src/lib/translations/bn/home.json (hunk 2 above)
Comment (tokitouq) [reply src/lib/translations/bn/home.json]: Yea Idk bengali, well resolved suggested changes on https://github.com/baseplate-admin/coreproject-v4-ui/pull/65/commits/24f7dc4f44f71081edc4b4e90e6dc5def08ed6e2

**Comment by tokitouq** [comment]
@baseplate-admin
The goal of this PR has achieved :tada: so Imma merge this for now.
lets work on translations on separate PR

**Pull request merged**